x/avs/keeper/avs.go

```diff
@@ -0,0 +1,60 @@
+package keeper
+
+import (
+	errorsmod "cosmossdk.io/errors"
+	sdkmath "cosmossdk.io/math"
+	"fmt"
+	"github.com/ExocoreNetwork/exocore/x/avs/types"
+
+	"github.com/cosmos/cosmos-sdk/store/prefix"
+	sdk "github.com/cosmos/cosmos-sdk/types"
+)
+
+func (k *Keeper) GetAVSSupportedAssets(ctx sdk.Context, avsAddr string) ([]string, error) {
```

Comment:
Please reuse the function `GetAVSInfo` to get avs info, and return res.AssetId then

Reply:
fixed